odl/tomo/geometry/detector.py

```diff
@@ -240,7 +240,7 @@ def __init__(self, params, detector_axis, grid=None):
 
     @property
     def detector_axis(self):
-        """The directions of the principial axes of the detector."""
+        """The directions of the principal axes of the detector."""
```

Comment:
I guess it's a copy-and-paste error. It's the 1d detector and it should be singular.